spec/koans.js

```diff
@@ -301,29 +302,29 @@ describe('destructuring objects. ', () => {
 describe('destructuring can also have default values. ', () => {
 
   it('for an empty array', () => {
-    const [a] = [];
-    //expect(a).toEqual(1)
+    const [a=1] = [];
+    expect(a).toEqual(1)
   });
 
   it('for a missing value', () => {
-    const [a,b,c] = [1,,3];
-    //expect(b).toEqual(2);
+    const [a,b=2,c] = [1,,3];
+    expect(b).toEqual(2);
   });
 
   it('in an object', () => {
-    const [a, b] = [{a: 1}];
-    //expect(b).toEqual(2);
+    const [a, b=2] = [{a: 1}];
+    expect(b).toEqual(2);
```

Comment:
Should be 

const [{a, b=2}] = [{a: 1}]
Otherwise a is equal to the whole object